activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -992,7 +992,7 @@ def assume_migrated_upto_version(version, migrations_paths)
           if (duplicate = inserting.detect {|v| inserting.count(v) > 1})
             raise "Duplicate migration #{duplicate}. Please renumber your migrations to resolve the conflict."
           end
-          execute "INSERT INTO #{sm_table} (version) VALUES #{inserting.map {|v| '(#{v})'}.join(', ') }"
+          execute "INSERT INTO #{sm_table} (version) VALUES #{inserting.map {|v| "(#{v})"}.join(', ') }"
```

Comment:
Maybe the coffee hasn't settled in yet but looking at a previous line I think we need to still quote the value.

`ActiveRecord::SchemaMigration.order('version').pluck(:version).map {|v| "('#{v}')" }.join(', ')`

Does the migration work if you add single quotes inside like this:
`execute "INSERT INTO #{sm_table} (version) VALUES #{inserting.map {|v| "('#{v}')"}.join(', ') }"`